lutris/sysoptions.py

```diff
@@ -184,7 +186,9 @@ def get_vk_icd_choices():
             choices.append(("AMDVLK Open source", amdvlk_files))
     if amdvlkpro_files:
         choices.append(("AMDGPU-PRO Proprietary", amdvlkpro_files))
-    choices.append((_("Unspecified (Use System Default)"), ""))
```

Comment:
This is gone as it'll now show `Auto: (your default GPU)` so it actually tells you what the default is.